README.md

```diff
@@ -1,349 +1,449 @@
-- [Memcached Store](https://npmjs.org/package/rate-limit-memcached)
-- [Mongo Store](https://www.npmjs.com/package/rate-limit-mongo)
-- [Precise Memory Store](https://www.npmjs.com/package/precise-memory-rate-limit) - similar to the built-in memory store except that it stores a distinct timestamp for each IP rather than bucketing them together.
+> Note: this module does not share state with other processes/servers by
+> default. If you need a more robust solution, I recommend using an external
+> store. See the [`stores` section](#store) below for a list of external stores.
 
 ### Alternate Rate-limiters
 
-This module was designed to only handle the basics and didn't even support external stores initially. These other options all are excellent pieces of software and may be more appropriate for some situations:
+This module was designed to only handle the basics and didn't even support
+external stores initially. These other options all are excellent pieces of
+software and may be more appropriate for some situations:
 
 - [rate-limiter-flexible](https://www.npmjs.com/package/rate-limiter-flexible)
 - [express-brute](https://www.npmjs.com/package/express-brute)
 - [rate-limiter](https://www.npmjs.com/package/express-limiter)
 
 ## Install
 
+From the npm registry:
+
+```sh
+# Using npm
+> npm install express-rate-limit
+# Using yarn or pnpm
+> yarn/pnpm add express-rate-limit
+```
+
+From Github Releases:
+
 ```sh
-$ npm install --save express-rate-limit
+# Using npm
+> npm install https://github.com/nfriedly/express-rate-limit/releases/download/v{version}/express-rate-limit.tgz
+# Using yarn or pnpm
+> yarn/pnpm add https://github.com/nfriedly/express-rate-limit/releases/download/v{version}/express-rate-limit.tgz
 ```
 
+Replace `{version}` with the version of the package that you want to your, e.g.:
+`6.0.0`.
+
+> **This library (v6.0.0 or greater) is now pure ESM**. CommonJS packages will
+> not be able to synchronously import this package, and it is recommended to use
+> the package with NodeJS version 12 or greater. Take a look at this
+> [article](https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c)
+> for more details.
+
 ## Usage
 
 For an API-only server where the rate-limiter should be applied to all requests:
 
 ```js
-const rateLimit = require("express-rate-limit");
+import rateLimit from 'express-rate-limit'
 
-// Enable if you're behind a reverse proxy (Heroku, Bluemix, AWS ELB or API Gateway, Nginx, etc)
+// Enable if you're behind a reverse proxy (Heroku, Bluemix, AWS ELB, Nginx, etc)
 // see https://expressjs.com/en/guide/behind-proxies.html
 // app.set('trust proxy', 1);
 
 const limiter = rateLimit({
-  windowMs: 15 * 60 * 1000, // 15 minutes
-  max: 100 // limit each IP to 100 requests per windowMs
-});
+	windowMs: 15 * 60 * 1000, // 15 minutes
+	max: 100, // Limit each IP to 100 requests per `window` (here, per 15 minutes)
+	standardHeaders: true, // Return rate limit info in the `RateLimit-*` headers
```

Comment:
Lets put both `standardHeaders` and `legacyHeaders` options in here. Just enabling the standard headers without disabling the legacy headers will mean both get sent.

```suggestion
	standardHeaders: true, // Return rate limit info in the `RateLimit-*` headers
	legacyHeaders: false, // Disable the `X-RateLimit-*` headers
```